docker-compose.review.yml

```diff
@@ -29,6 +29,7 @@ services:
       - .:/srv/vets-api/src:cached
       - dev_bundle:/usr/local/bundle
       - ../.secret:/srv/vets-api/secret:cached
+      - ../.pki:/srv/vets-api/pki:cached
```

Comment:
So this is temporary? Am I understanding that correctly?

Reply:
this particular change is permanent. It's to let vets-api access certs we provide for authentication to backend services. The other cert stuff is for CA cert store (and accessed through /etc/ssl)